agent/consul/rpc.go

```diff
@@ -37,11 +37,11 @@ const (
 	// If it's over 1MB something is probably being abusive.
 	raftWarnSize = 1024 * 1024
 
-	// enqueueLimit caps how long we will wait to enqueue
+	// defaultRaftApplyTimeout caps how long we will wait to enqueue
 	// a new Raft command. Something is probably wrong if this
 	// value is ever reached. However, it prevents us from blocking
 	// the requesting goroutine forever.
-	enqueueLimit = 30 * time.Second
+	defaultRaftApplyTimeout = 30 * time.Second
```

Comment:
Since the default from agent/config/default.go is no being used, I don't think we need this one anymore.

Reply:
ah I missed the overwriting in DefaultConfig(), thanks for pointing.